discovery-provider/src/utils/web3_provider.py

```diff
@@ -28,13 +27,14 @@ def get_web3(web3endpoint=None):
     if not web3endpoint:
         # attempt local rpc, check if healthy
         try:
-            if requests.get(LOCAL_RPC + "/health").status_code == 200:
-                web3endpoint = LOCAL_RPC
-                logger.info("web3_provider.py | using local RPC")
+            primary_acdc_rpc = os.getenv("audius_web3_primary_host")
```

Comment:
it looks like ordering of this string is incorrect in audius docker compose
https://github.com/AudiusProject/audius-docker-compose/pull/307/files#diff-4b69066f9a272aaef7a06adf061c1ce950c8933625465055adbd332d57f03690R39

Reply:
`audius_primary_web3_host` != `audius_web3_primary_host `, secondary has the same misspelling

Reply:
i was just updating docker compose